src/Parameters.hs

```diff
@@ -0,0 +1,31 @@
+module Parameters (
+    Parameters(..),
+    argParser,
+    paramInfo
+) where
+
+import Options.Applicative
+
+newtype Parameters = Parameters
+  { config :: FilePath }
+
+argParser :: Parser Parameters
+argParser = Parameters
+        <$> strOption
+          ( long "config"
+          <> metavar "FILEPATH"
+          <> value "config.toml"
+          <> help "filepath where config is loaded from")
```

Comment:
Maybe mconcat from Data.Monoid  will be better.

Reply:
I think the mappend operator is more idiomatic in this case.